README.md

```diff
@@ -0,0 +1,182 @@
+# Liquid Crystal Library for Arduino
+
+## Description
+
+This library allows an Arduino board to control LiquidCrystal displays (LCDs) based on the Hitachi HD44780 (or a compatible) chipset, which is found on most text-based LCDs. The library works within either 4- or 8-bit mode (i.e. using 4 or 8 data lines in addition to the rs, enable, and, optionally, the rw control lines).
+
+## Installation
+
+![image](https://user-images.githubusercontent.com/36513474/67681689-a5eee280-f9af-11e9-837d-350f8f0aebd0.png)
+
+### First Method
+
+1. In the Arduino IDE, navigate to Sketch > Include Library > Manage Libraries
+1. Then the Library Manager will open and you will find a list of libraries that are already installed or ready for installation.
+1. Then search for LiquidCrystal using the search bar.
+1. Click on the text area and then select the specific version and install it.
+
+### Second Method
+
+1. Navigate to the Releases page.
+1. Download the latest release.
+1. Extract the zip file
+1. In the Arduino IDE, navigate to Sketch > Include Library > Add .ZIP Library
+
+## Features
+
+- ### Easy to use
+
+    It is simple, basic and very easy to understand. The user only needs to import the library in his code and start using its functions.
+
+- ### Compatible chipset
+
+    ![image](https://user-images.githubusercontent.com/36513474/67893895-4e03d780-fb79-11e9-814b-d8fa03689344.png)
+
+    Using Liquid Crystal library, Arduino board can control liquid displays based on the Hitachi HD44780 chipset. These chipset LCDs are easily available, come in different shapes and sizes.
+
+- ### No technical details required
+
+    A key benefit of using this liquid crystal library is that there is no need to care that much about the technical details or timings when using the display. The library happily does all of this.
+
+- ### Extendable
+
+    The library is extendable, therefore it can grow to support new LCD control devices such as I2C, SPI, Serial, 1wire, RF, CAM, etc.
+
+- ### Intuitive syntax
+
+    Liquid Crystal has a simple and intuitive syntax to handle variables and functions. The user only needs to import the library and calls its functions
+
+- ### Give back
+
+    Liquid Crystal is free for everyone. The licensed library can be copied, redistributed and used in the projects, assignments or anywhere.
+
+## Functions
+
+- LiquidCrystal()
```

Comment:
I don't find this section useful at all, the website has substantially more information.